src/control-plane/backend/lambda/batch-insert-ddb/index.ts

```diff
@@ -76,7 +78,16 @@ export const handler = async (
 async function _handler(event: CdkCustomResourceEvent) {
   const requestType = event.RequestType;
 
-  if (requestType == 'Create' || requestType == 'Update') {
+  if (requestType == 'Create') {
+    await cleanData(event);
+    await batchInsert(event);
+  } else if (requestType == 'Update') {
+    const lastModifiedTime = event.ResourceProperties.lastModifiedTime;
+    const oldLastModifiedTime = (event as CloudFormationCustomResourceUpdateEvent).OldResourceProperties.lastModifiedTime;
```

Comment:
Do you have a case for upgrading from a version without `lastModifiedTime` field?